spec/features/likes_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.feature 'Likes', type: :feature do
+
+  let(:user) { FactoryBot.create(:user, :with_posts) }
+  let(:other_user) { FactoryBot.create(:user) }
+
+  before do
+    visit root_path
+    fill_in 'user_email',    with: other_user.email
+    fill_in 'user_password', with: other_user.password
+    click_button 'Log in'
+  end
+
+  scenario "user can like other user posts" do
```

Comment:
This test is failing.
Need to address before I can give you the 👍 
[REQUIRED]

Reply:
### Hi @mcrd25, :raising_hand_man: 

Thanks for the review, :bowing_man: :+1: 
The test is not failing on the last commit. :thinking: 
Can you send the test result to reproduce this issue?

See the test results below:

``` sh
$ rspec

CommentsController
  #create
    as an authenticated user
      comment on a post
    as a guest
      returns a 302 response
      redirects to the sign-in page
  #destroy
    as an authenticated user
      delete a comment
    as a guest
      returns a 302 response
      redirects to the sign-in page

LikesController
  #create
    as an authenticated user
1
4639
2
      like on a post
    as a guest
      returns a 302 response
      redirects to the sign-in page
  #destroy
    as an authenticated user
      delete a like
    as a guest
      returns a 302 response
      redirects to the sign-in page

PostsController
  #index
    as an authenticated user
      responds successfully
      returns a 200 response
    as a guest
      redirects to the sign-in page
      returns a 302 response
  #create
    as an authenticated user
      adds a post
    as a guest
      returns a 302 response
      redirects to the sign-in page

Comments
  user can write comments on posts
  user can delete comments
  user can't delete other user comments

Likes
  user can like and unlike posts

Posts
  user sign in and creates a new post

Users
  create a new user
  log in and user

Comment
  is invalid without a user
  is invalid without a post
  is invalid without a content

Like
  is invalid without a user
  is invalid without a post

Post
  is invalid without a content
  is invalid without a user

User
  is valid with a name, email, and password
  is invalid without a name
  is invalid without an email address
  is invalid with a duplicate email address

Finished in 1.71 seconds (files took 0.99723 seconds to load)
37 examples, 0 failures
```